README.md

```diff
@@ -7,38 +7,83 @@ It is provided as a BSD-license package, hosted on Github.
+The result is a file (or a byte buffer) called "dictionary", which can be loaded before compression and decompression.
+By using this dictionary, the compression ratio achievable on small data improves dramatically :
+
+| Collection Name    | Direct compression | Dictionary Compression | Gains  | Average unit | Range       |
+| ---------------    | ------------------ | ---------------------- | -----  | ------------:| -----       |
+| Small JSON records | x1.331 - x1.366	  | x5.860 - x6.830        | ~ x4.7 | 300          | 200 - 400   |
+| Mercurial events   | x2.322 - x2.538    | x3.377 - x4.462        | ~ x1.5 | 1.5 KB       | 20 - 200 KB |	
+| Large JSON docs    | x3.813 - x4.043    | x8.935 - x13.366       | ~ x2.8 | 6 KB         | 800 - 20 KB |	
+
+It has to be noted that these compression gains are achieved without any speed loss, and even some faster decompression processing.
+
+Dictionary work if there is some correlation in a family of small data (there is no _universal dictionary_).
+Hence, deploying one dictionary per type of data will provide the greater benefits.
+
+Large documents will benefit proportionally less, since dictionary gains are mostly effective in the first few KB.
+Then there is enough history to build upon, and the compression algorithm can rely on it to compress the rest of the file.
 
-Zstd has not yet reached "stable format" status. It doesn't guarantee yet that its current compressed format will remain stable and supported in future versions. During this period, it can still change to adapt new optimizations still being investigated. "Stable Format" is projected sometimes early 2016. 
 
-That being said, the library is now fairly robust, able to withstand hazards situations, including invalid inputs. The library reliability has been tested using [Fuzz Testing](https://en.wikipedia.org/wiki/Fuzz_testing), with both [internal tools](programs/fuzzer.c) and [external ones](http://lcamtuf.coredump.cx/afl). Therefore, it seems now safe to test Zstandard even within production environments.
+### Status
+
+Zstd has not yet reached "stable format" status. It doesn't guarantee yet that its current compression format will remain stable in future versions. During this period, it can still change to adapt new optimizations still being investigated. "Stable Format" is projected H1 2016, and will be tagged `v1.0`.
+
+That being said, the library is now fairly robust, able to withstand hazards situations, including invalid inputs. It also features legacy support, so that documents compressed with current and previous version of zstd can still be decoded in the future. 
+Library reliability has been tested using [Fuzz Testing](https://en.wikipedia.org/wiki/Fuzz_testing), with both [internal tools](programs/fuzzer.c) and [external ones](http://lcamtuf.coredump.cx/afl). Therefore, Zstandard is not considered safe for testings, even within production environments.
```

Comment:
This was a typo (from "no**w**" to "no**t**").
It's supposed to be already corrected...